DNN Platform/DotNetNuke.Abstractions/Portals/IPortalAliasInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿// 
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
This header is not correct.  Proper header format is

// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

Per guidance here: https://github.com/dotnet-foundation/foundation/blob/master/guidance/copyright.md